### ISSUE TITLE
[Bug]: editing markers or hotspots breaks UI

### ISSUE THREAD

**Issue by passioneight** [pimcore/pimcore#12777]
### Expected behavior

One can edit markers or hotspots without any issues.

### Actual behavior

UI breaks and needs reload to recover.

### Steps to reproduce

1. Open any document with a `pimcore_image`, for example, [this one](https://demo.pimcore.com/admin/login/deeplink?document_46_page)
3. Add a marker or hotspot to the image (see screenshot)
4. Reload the document
5. Click "Add data" on the marker/hotspot again (see second screenshot) 
6. Reload UI to recover

![image](https://user-images.githubusercontent.com/57256630/181238155-09a25ffa-8269-4742-8508-377218342c9a.png)

![image](https://user-images.githubusercontent.com/57256630/181238440-d0320141-b1cc-4257-b7d7-c94eadc3b679.png)

**Comment by dvesh3** [comment]
@passioneight so you are not saving the document after step 2? or you have auto save enabled?

**Comment by passioneight** [comment]
@dvesh3 this is the Pimcore Demo. Drafts are enabled and it does not matter if I only reload or save and reload.

**Comment by aryaantony92** [comment]
Fixed by #12802

**Comment by passioneight** [comment]
I can confirm, that the dialog can now be opened. However, when using assets, the path is lost, instead the ID is set.

Reproduce like this:

1. Add Asset relation in marker/hotspot
2. Drag & Drop Asset onto relation field
![image](https://user-images.githubusercontent.com/57256630/183019498-cbe40d41-0a33-41ac-8804-b2642364d3e1.png)
3. Save everything an reload the document
4. Reopen the marker/hotspot and find the an ID is now displayed of the path
<img width="540" alt="image" src="https://user-images.githubusercontent.com/57256630/183019032-f25e0a52-2483-4cd5-9080-942f48e5e1ff.png">

**Comment by dvesh3** [comment]
> I can confirm, that the dialog can now be opened. However, when using assets, the path is lost, instead the ID is set.
> 
> Reproduce like this:
> 
> 1. Add Asset relation in marker/hotspot
> 2. Drag & Drop Asset onto relation field
>    ![image](https://user-images.githubusercontent.com/57256630/183019498-cbe40d41-0a33-41ac-8804-b2642364d3e1.png)
> 3. Save everything an reload the document
> 4. Reopen the marker/hotspot and find the an ID is now displayed of the path
> 
> <img alt="image" width="540" src="https://user-images.githubusercontent.com/57256630/183019032-f25e0a52-2483-4cd5-9080-942f48e5e1ff.png">

@passioneight could you please create a separate issue for this bug? thanks!
